test-matrix/table.md

```diff
@@ -1,35 +1,64 @@
+
+
 # Import behavior matrix
+
 What is available as a result of `import * as namespace from "x.cjs"`
-| | 1.cjs:<br>default | 1.cjs:<br>no | 1.cjs:<br>even | 1.cjs:<br>default<br>.even |  2.cjs:<br>default | 2.cjs:<br>no | 2.cjs:<br>even | 2.cjs:<br>default<br>.even |  3.cjs:<br>default | 3.cjs:<br>no | 3.cjs:<br>even | 3.cjs:<br>default<br>.even |  4.cjs:<br>default | 4.cjs:<br>no | 4.cjs:<br>even | 4.cjs:<br>default<br>.even |  5.cjs:<br>default | 5.cjs:<br>no | 5.cjs:<br>even | 5.cjs:<br>default<br>.even
-| --- | --- | --- | --- | --- |  --- | --- | --- | --- |  --- | --- | --- | --- |  --- | --- | --- | --- |  --- | --- | --- | ---
-| node | ✔️ | ✔️ | ✔️ | ✔️ |   ✔️ | ✔️ | ❌ | ✔️ |   ✔️ | ✔️ | ❌ | ✔️ |   ✔️ | ✔️ | ✔️ | ✔️ |   ✔️ | ❌ | ❌ | ✔️
-| endo | ✔️ | ✔️ | ✔️ | ✔️ |   ✔️ | ✔️ | ❌ | ✔️ |   ✔️ | ✔️ | ❌ | ✔️ |   ✔️ | ✔️ | ✔️ | ✔️ |   ✔️ | ❌ | ❌ | ✔️
-| webpack | ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️
-| rollup | ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️ |   ✔️ | ❌ | ✔️ | ✔️
-| parcel | ❌ | ❌ | ✔️ | ❌ |   ❌ | ❌ | ✔️ | ❌ |   ❌ | ❌ | ✔️ | ❌ |   ❌ | ❌ | ✔️ | ❌ |   ❌ | ❌ | ✔️ | ❌
+
+|  | node | endo | webpack | rollup | parcel | esbuild | typescript | TS esModuleInterop |
```

Comment:
Might be good (and also clearly much harder) to compare Node.js 12 vs >=14

Reply:
With a bunch of external dependencies, pulling in fnm and installing versions I got a table with node 12 14 16 giving the same results. Boring.

Reply:
Huh. There are some differences. It would be neat to isolate the cases, but not in scope for this change.